### PR TITLE
Fix instructions for enabling and disabling debug logging

### PR DESCRIPTION
**Pull request by jakubka**
The instructions were out of date with how the UI looks at the moment

## Proposed change

Fixing out of date information about debug logging.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

I made some screenshots from my instance. Not sure if they are formatted correctly, but I haven't found any standards regarding that.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration/troubleshooting.markdown [hunk 1]
```diff
@@ -104,22 +104,22 @@ The first thing you will need before reporting an issue online is debug logs and
 
 ### Enabling Debug Logging
 
-To enable debug logging for an integration, go to **Settings** > **Devices & Services** and select the integration. Select the triple dots for the integration giving you trouble and select **Enable Debug Logging**.
-
-### Disable Debug Logging and Download Logs
-
-Once you enable debug logging, you ideally need to make the error happen. Run your automation, change up your device or whatever was giving you an error and then come back and disable Debug Logging. Disabling debug logging is the same as enabling, but now you will see **Disable Debug Logging**. After you disable debug logging, it will automatically prompt you to download your log file. Save this to a safe location to upload later.
+To enable debug logging for an integration, go to **Settings** > **Devices & Services** > **Integrations** and go to the detail page of the integration. Click the **Enable Debug Logging** button on the left side of the integrationd detail page.
```
Comment (frenck) [inline source/_docs/configuration/troubleshooting.markdown]:
```suggestion
To enable debug logging for an integration, go to **Settings** > **Devices & Services** > **Integrations** and go to the detail page of the integration. Select the **Enable Debug Logging** button on the left side of the integration detail page.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @jakubka 👍 

../Frenck

**Review by frenck — APPROVED** [review]
Wrong button, let me try again 👍 

../Frenck

**Comment by jakubka** [comment]
Thanks for noticing and fixing the typo @frenck 👍
